src/api/server.ts

```
@@ -49,23 +53,67 @@ app.get(
             : (req.query.section as string),
       };
 
-      const offset = (page - 1) * limit;
+      // Generate cache key
+      const cacheKey = generateLeaderboardCacheKey(sortBy, page, limit, filters);
```

ESLint

[error] Replace `sortBy,·page,·limit,·filters` with `⏎········sortBy,⏎········page,⏎········limit,⏎········filters,⏎······`

(prettier/prettier)